ppb/systems/renderer.py

```diff
@@ -233,14 +248,35 @@ def prepare_resource(self, game_object):
 
         surface = image.load()
         try:
-            return self._texture_cache[surface]
+            texture = self._texture_cache[surface]
         except KeyError:
             texture = SmartPointer(sdl_call(
                 SDL_CreateTextureFromSurface, self.renderer, surface,
                 _check_error=lambda rv: not rv
             ), SDL_DestroyTexture)
             self._texture_cache[surface] = texture
-            return texture
+
+        opacity = getattr(game_object, 'opacity', 255)
+        opacity_mode = getattr(game_object, 'opacity_mode', flags.BlendModeBlend)
+        opacity_mode = OPACITY_MODES[opacity_mode]
+        color = getattr(game_object, 'color', (255, 255, 255))
```

Comment:
```suggestion
        color = getattr(game_object, 'tint', (255, 255, 255))
```

Regarding suggestion on discord.